aws/sdk-codegen/src/main/kotlin/software/amazon/smithy/rustsdk/AwsCrateDocsDecorator.kt

```diff
@@ -103,12 +103,13 @@ internal class AwsCrateDocGenerator(private val codegenContext: ClientCodegenCon
             else -> rawTemplate(text + "\n", *args)
         }
 
-    private fun docText(
+    internal fun docText(
         includeHeader: Boolean,
         includeLicense: Boolean,
         asComments: Boolean,
     ): Writable = writable {
         val moduleName = codegenContext.settings.moduleName
+        val stableVersion = !codegenContext.settings.moduleVersion.startsWith("0.")
```

Comment:
do we want to assert it _does_ start with a number or something? I'm 99% it's never `v...` but I wouldn't want to accidentally ship 1.0 in case it did.

Reply:
Good call.